pkg/authenticator/authenticator.go

```diff
@@ -142,6 +142,7 @@ func (auth *Authenticator) Login() error {
 
 		return log.RecordedError(log.CAKC012E, err.Error())
 	}
+	log.InfoLogger.Printf(log.CAKC015I, auth.Config.ClientCertPath)
```

Comment:
Are we ok with this, securitywise?
Maybe we can put just the last folder?

I would also consider using this only in DEBUG mode. Doesn't feel like a business information but something you would want to see during deployment only

Reply:
1. this field is configured by the user but i guess we can get almost the same value without printing the path and just printing that we loaded the certificate successfully. wdyt?
2. we don't have a debug log in the authn-client atm so i don't want to have a whole effort of adding such a logger, an env var, etc. We don't have many logs so it doesn't add a lot of noise.

Reply:
1 & 2: LGTM thanks

Reply:
Out of curiosity, wdym it's configurable? We have an open issue (afaik) in conjur to make it configurable, but last I looked its hardcoded in the server

Reply:
Also, I wish we had a debug log or configurable levels. These log lines are already really noisy in secretless and this PR is adding multiple new lines that will repeat as the authn cycle continues.

Reply:
> Out of curiosity, wdym it's configurable? We have an open issue (afaik) in conjur to make it configurable, but last I looked its hardcoded in the server

@izgeri you are correct. it is configurable only in the client (so it doesn't give any value). The point was that it is not a sensitive data as it is configurable in the env.

Reply:
> Also, I wish we had a debug log or configurable levels. These log lines are already really noisy in secretless and this PR is adding multiple new lines that will repeat as the authn cycle continues.

we actually have that for the secrets provider and can get it done here as well. I wouldn't have it in the current effort but I'll create an issue for it.